tests/test_TFNetworkLayer.py

```diff
@@ -1078,3 +1126,50 @@ def make_feed_dict(seq_len=10):
     #if len(list(threading.enumerate())) > 1:
     #  print("Warning, more than one thread at exit:")
     #  better_exchook.dump_all_thread_tracebacks()
+  with make_scope() as session:
```

Comment:
I think this ended up in the wrong function?
Please sanity check the code/changes you actually committed.